components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -1237,7 +1237,14 @@ public String exportSPApplicationFromAppID(String applicationId, boolean exportS
                     "for the provided id: " + applicationId);
         }
         String appName = application.getApplicationName();
-        return exportSPApplication(appName, exportSecrets, tenantDomain);
+        String exportedApp;
+        try {
+            startTenantFlow(tenantDomain);
+            exportedApp = exportSPApplication(appName, exportSecrets, tenantDomain);
```

Comment:
Wouldn't it be better to return the exportSPApplication(appName, exportSecrets, tenantDomain) result here?
The variable does not serve any purpose AFAIS.